.github/workflows/build-and-upload-wheel.yaml

```diff
@@ -30,12 +30,7 @@ jobs:
       - name: Set up python 3.7
         uses: actions/setup-python@v4
         with:
-          python-version: '3.7'
-      - name: Install Protoc
-        uses: arduino/setup-protoc@v1
-        with:
-          version: '3.17.3'
-          repo-token: ${{ secrets.FETCH_RELEASE_DATA }}
+          python-version: 3.7
```

Comment:
Shall we support different python versions here as well?

Reply:
Since we don't have any compiled binary, the built wheel is compatible with all Python3 versions. 

And we run examples against the built wheel in our nightly build with different Python versions.